product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx

```diff
@@ -4,29 +4,33 @@ title: "Backing up and restoring"
 
 ## Backups 
 
-BigAnimal backs up the data in your PostgreSQL clusters. Depending on the cloud your cluster is deployed on, BigAnimal uses either of the following cloud object storage solutions:
+BigAnimal backs up the data in your PostgreSQL clusters. You can change the retention period on the [Backups tab](../getting_started/creating_a_cluster/#additional-settings-tab) when you create or edit your cluster. Depending on the cloud your cluster is deployed on, BigAnimal uses either of the following cloud object storage solutions:
 
 | Cloud | Object storage solution                                                        |   |
 |----------------------------|--------------------------------------------------------------------------------------|---|
 | AWS                      | Amazon S3  (standard tier)                                                                          |   |
 | Azure                        | Azure Blob Storage (in the "hot" access tier with geo-zone-redundant storage (GZRS)) |   |
 
+Your organization is responsible for the charges associated with the cloud object storage solution.
 
-!!! Note
-    Backups in AWS do not currently support cross-regional replication.
+PostgreSQL clusters in BigAnimal are continuously backed up through a combination of base backups and transaction log (WAL) archiving. When a new cluster is created, an initial "base" backup is taken. After that, every time a WAL file is closed, which is, by default, up to every 5 minutes, it is automatically uploaded to the cloud object storage solution. 
 
-PostgreSQL clusters in BigAnimal are continuously backed up through a combination of base backups and transaction log (WAL) archiving. When a new cluster is created, an initial "base" backup is taken. After that, every time a WAL file is closed, which is, by default, up to every 5 minutes, it is automatically uploaded to the cloud object storage solution. Your organization is responsible for the charges associated with the cloud object storage solution.
+## Log shipping
 
- You can change the retention period on the [Additional Settings tab](../getting_started/creating_a_cluster/#additional-settings-tab) when you create or edit your cluster. 
 
+As new data is archived, BigAnimal copies and transfers the WAL files from your cloud object storage solution and *asynchronously* transfers them to faraway replicas. If the time interval set for closing a WAL file is too large, it can introduce a delay until the faraway replica receives the latest record. See [Backups](/biganimal/latest/using_cluster/04_backup_and_restore/#backups).
 
-    
+
+Log shipping does not make any changes to the database tables. It facilitates:
+
+- Low administration overhead
+- Predictable timelines
+- Minimal network requirements
 
 ## Restores
```

Comment:
is it possible to mention in this section (or another relevant places) that restores are also possible for faraway replicas? Restoring a replica will create a new standalone cluster

Reply:
Added this detail